### PR TITLE
BP-2794-iDEAL-Pay-by-bank-is-broken

### PR DESCRIPTION
**Pull request by vegimcarkaxhija**
fix Ideal and Credit cart not showing 5 issuers in begging, make images smaller, change the color of the background

### CODE REVIEW

File: views/js/buckaroo.js [hunk 1]
```diff
@@ -265,65 +265,69 @@ $(document).ready(function () {
...
     static SHOW_MORE_BANKS = 5;
-    listen() {
-        this.toogleMethods();
 
-        jQuery("body").on("updated_checkout", function () {
-            self.initMethod()
-        });
+    listen() {
+        this.toggleMethods();
+        jQuery("body").on("updated_checkout", () => this.initMethod());
```
Comment (harli91) [inline views/js/buckaroo.js:275]: whats the use of this event listener?

File: views/js/buckaroo.js [hunk 2]
```diff
@@ -265,65 +265,69 @@ $(document).ready(function () {
...
 
-        this.getElementToToggle().toggle(isDown);
+        toggle.toggleClass('bk-toggle-down bk-toggle-up');
+    }
+
+    hideExcessIssuers(selector) {
+        const isPayByBank = selector.hasClass('bk-paybybank-selector');
+        const selectedIssuer = isPayByBank ? selector.find('input:checked') : null;
+
+        if (isPayByBank && selectedIssuer.length) {
```
Comment (harli91) [inline views/js/buckaroo.js:306]: what happens if selectedIssuer is null ? null.length evaluates to ?

File: views/js/buckaroo.js [hunk 3]
```diff
@@ -265,65 +265,69 @@ $(document).ready(function () {
...
+            selector.children().not(selectedIssuer.closest('.bk-method-issuer')).hide();
+        } else {
+            selector.children(`:nth-child(n+${BuckarooCheckout.SHOW_MORE_BANKS})`).hide();
+        }
     }
 
     initMethod() {
-        this.getElementToToggle().hide();
+        jQuery('.bk-method-selector').each((_, elem) => {
+            const selector = jQuery(elem);
```
Comment (harli91) [inline views/js/buckaroo.js:315]: elem should be a jquery object already

File: views/js/buckaroo.js [hunk 4]
```diff
@@ -265,65 +265,69 @@ $(document).ready(function () {
...
-                this.getElementToToggle().show();
-                jQuery('.bk-toggle-text').text(jQuery('.bk-toggle-text').attr('text-less'));
-            } else {
-                jQuery('.bk-toggle-wrap').show();
-            }
+            const isDown = jQuery('.bk-toggle-down').length;
+            jQuery('.bk-toggle-wrap').toggle(!isDown);
+            jQuery('.bk-toggle-down').toggleClass('bk-toggle-up bk-toggle-down');
+            jQuery('.bk-method-selector').children().show();
+            this.toggleTextElements.text(this.toggleTextElements.attr('text-less'));
```
Comment (harli91) [inline views/js/buckaroo.js:330]: here I think you need a each function....

**Review by harli91 — CHANGES_REQUESTED** [review]

File: views/js/buckaroo.js (hunk 3 above)
Comment (vegimcarkaxhija) [reply views/js/buckaroo.js:315]: When using each function in jQuery, the provided element (elem in this case) is a DOM element, not a jQuery object.

**Review by harli91 — APPROVED** [review]

**Pull request merged**